### PR TITLE
Split skia download/unzipping task creation and invocation

### PR DESCRIPTION
**Pull request by Schahen**
This commit is first step in set of actions we need to take to build skia-package via gradle in the skiko repository

However it makes sense on it's own as well.

### CODE REVIEW

File: skiko/build.gradle.kts [hunk 1]
```diff
@@ -12,6 +14,47 @@ plugins {
     id("org.gradle.crypto.checksum") version "1.4.0"
 }
 
+fun createDownloadTasks() {
```
Comment (eymar) [inline skiko/build.gradle.kts:17]: Is it possible to keep this function in buildSrc?  The motivation is to limit the growth of build.gradle.kts

Comment (Schahen) [reply skiko/build.gradle.kts]: Totally makes sense, renamed it to `declareSkiaTasks` and moved to the SkikoProjectContext (which belongs to buildSrc)

File: skiko/buildSrc/src/main/kotlin/properties.kt [hunk 2]
```diff
@@ -162,7 +156,7 @@ class SkikoProperties(private val myProject: Project) {
                 System.getenv()["SKIA_DIR"]
                 ?: System.getProperty("skia.dir")
                 ?: myProject.findProperty("skia.dir")?.toString()
-            )?.let { File(it) }?.takeIf { it.isDirectory }
+            )?.let { File(it) }
```
Comment (eymar) [inline skiko/buildSrc/src/main/kotlin/properties.kt:159]: Was this check for `isDirectory` removed intentionnaly?

**Review by eymar — APPROVED** [review]

File: skiko/buildSrc/src/main/kotlin/properties.kt (hunk 2 above)
Comment (Schahen) [reply skiko/buildSrc/src/main/kotlin/properties.kt]: Without validation  there was always a possibility to just have a seemingly valid local build which in actuality was remote build. I  totally agree it's better to keep it in getter. Pushed

**Pull request merged**